aggie-coding-club/Rev-Registration
# Basic options default to no preference

## Description

Default for BasicOptions is now No Preference instead of Exclude. Also added test.

## Rationale

We attempted to make this change a while ago in #345, but there are multiple places that reference a default value for basic options, and one of them was missed.

## How to test

I created a test to verify that the change worked. The test failed when I first wrote it, but passed after I made the change in `BasicOptionsRow.tsx`. You can also check in browser to see that it works.

## Screenshots

|Before|After|
|--|--|
|![image](https://user-images.githubusercontent.com/10082177/107888980-ba712a00-6ed5-11eb-8d18-daac57ff125a.png)|![image](https://user-images.githubusercontent.com/10082177/107888897-07083580-6ed5-11eb-92d6-0178ff4fd8d7.png)|

## Related tasks

Closes #501
